apps/playground/src/app/modules/components/stepper-module/stepper-module.component.scss

```diff
@@ -1,13 +1,12 @@
 .simulation {
   display: flex;
-
-  .settings {
-    width: 50%;
-    background-color: indianred;
-  }
+  flex-direction: column;
 
   play-simulator {
-    width: 50%;
     background-color: aqua;
   }
 }
+
+.stepper-container {
+  width: 40%;
```

Comment:
Why do you constrain the width ? I thought you were trying to give it the possibility to take as much space as it can...